src/Tickets/Admin/Glance_Items.php

```diff
@@ -41,6 +41,8 @@ public function hooks() {
 	 * @return array $items The maybe modified array of items to be displayed.
 	 */
 	public function custom_glance_items_attendees( $items = [] ): array {
+		$items = is_array( $items ) ? $items : [];
```

Comment:
```suggestion
		$items = (array) $items;
```

Is a bit more clearer.

Reply:
AH I misinterpreted (this Slack thread)[https://lw.slack.com/archives/C01T0TSD4SC/p1695223619243249] that that was how we used to have it, but now I'm understanding that was Stephen's suggestion back in 2023 ;)